vnext/Shared/OInstance.cpp

```diff
@@ -551,7 +545,6 @@ std::vector<std::unique_ptr<NativeModule>> InstanceImpl::GetDefaultNativeModules
         return Microsoft::React::CreateHttpModule(transitionalProps);
       },
       nativeQueue));
-#endif
 
   modules.push_back(std::make_unique<CxxNativeModule>(
```

Comment:
If I'm reading this correctly, this change puts WebSocketModule into the `#if (defined(_MSC_VER) && !defined(WINRT))` where it wasn't before. Is this okay for Desktop?

Reply:
It is.
As of #11867, UWP uses the TurboModule variant for WebSockets, which would replace this entry in runtime anyway.

Putting it inside that conditional compilation makes it exclusive to Desktop.